### PR TITLE
[#54] Bearer token generation example

### PR DESCRIPTION
**Pull request by KirillovDenis**
Signed-off-by: Denis Kirillov <denis@nspcc.ru>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -364,6 +364,67 @@ some data and upload it via any available NeoFS HTTP Protocol Gateway by adding
...
 
+##### Bearer token generation
+In order to generate bearer token, you need to know container owner key and 
+public key of sender who will be do request to NeoFS (in our case it's gateway key).
+
+For example, we have
+**KxDgvEKzgSBPPfuVfw67oPQBSjidEiqTHURKSDL1R7yGaGYAeYnr** (private key) and 
+**03b69ba58cce4923af13ecff10702e59e4ad4560d037edda34d19f45826578101f** (public key)
+
+We will use golang for generation. Firstly need import the following packages:
```
Comment (roman-khimov) [inline README.md:375]: Maybe we can write some JSON directly?

File: README.md [hunk 2]
```diff
@@ -364,6 +364,67 @@ some data and upload it via any available NeoFS HTTP Protocol Gateway by adding
 the corresponding header to the upload request. Accessing the ACL protected data
 works the same way.
 
+##### Bearer token generation
```
Comment (roman-khimov) [inline README.md:367]: example

**Review by roman-khimov — APPROVED** [review]

**Comment by roman-khimov** [comment]
Squash them please.

**Comment by roman-khimov** [comment]
And add some "closes #54" comment to auto-close it.

**Comment by KirillovDenis** [comment]
Closes #54

**Pull request merged**